initializr-generator/src/test/java/io/spring/initializr/metadata/DependencyTests.java

```diff
@@ -272,6 +272,21 @@ public void resolveMatchingWithCustomGroupId() {
 				"foo", "com.acme", "bar", "0.3.0.RELEASE");
 	}
 
+	@Test
+	public void resolveVersionRangeWithX() {
```

Comment:
Would this be better as two separate tests?

Reply:
yep, I'll split it.